webapp/channels/src/sass/components/_buttons.scss

```diff
@@ -71,6 +71,7 @@ button {
         &.btn-xs {
             width: 24px;
             height: 24px;
+            padding: 0;
```

Comment:
I wonder why this is needed. What is generally overwriting this value? Will this affect (in any unintended way) any of the other buttons thorough the product?

Reply:
Thanks for the explanation! I should have read the whole css file :sweat_smile: